test/k6/src/tests/orders_sms.js

```diff
@@ -196,31 +177,14 @@ function TC05_GetSmsNotificationSummary(data, orderId) {
  * 05 - GET sms notification summary
  */
 export default function (data) {
-  try {
-    if (data.runFullTestSet) {
-      var selfLink = TC01_PostSmsNotificationOrderRequest(data);
-      let id = selfLink.split("/").pop();
-      TC02_GetNotificationOrderById(data, selfLink, id);
-      TC03_GetNotificationOrderBySendersReference(data);
-      TC04_GetNotificationOrderWithStatus(data, id);
-      TC05_GetSmsNotificationSummary(data, id);
-    } else {
-      // Limited test set for use case tests
-      var selfLink = TC01_PostSmsNotificationOrderRequest(data);
-      let id = selfLink.split("/").pop();
-      TC05_GetSmsNotificationSummary(data, id);
-    }
-  } catch (error) {
-    addErrorCount(false);
-    throw error;
-  }
-}
+  var selfLink = TC01_PostSmsNotificationOrderRequest(data);
```

Comment:
think I prefer the old notation. If we get to a point where a TC 6 should be a part of the full test set but not the regression tests what do we do? 

It is duplication of code, but more explicit the order of the tests in each case?

Reply:
Added an else, but keeping the POST test on top for both scenario.